react-js/react-part-2.md

```diff
@@ -35,19 +33,71 @@ In introduction to [React](/react.md) we discussed about how React makes website
 
 ### Guided Practice
 
-1. Define state.
-  - >	``` state = {} ```
-2. Initialize state
+- Define state.
+  ``` 
+  state = {} 
+  ```
+- Initialize state
   - State in class component can be initialized in two ways
-    - Directly inside the 
+    - Directly inside the class component, outside all methods.
+      ``` 
+      state = {
+        message: "Hello world"
+      }
+      ```
+
+    - Inside class constructor
+      ```
+      constructor(){
+        this.state = {
+          message: "Hello world"
+          }
+      }
+      ```
 
-3. State should not be mutated directly i.e. a copy of state should be created and then updated. (already covered in lesson)
+- State should not be mutated directly i.e. a copy of state should be created and then updated. (already covered in lesson)
```

Comment:
I think the `(already covered...` is leftover from an older version. It has not already been covered so it's confusing. Please remove this.